_docs/quickstarts/scanner/descriptions.md

```diff
@@ -74,6 +90,20 @@ Inventory pipelines:
 
 - `load_forwarding_rules_pipeline`
 
+## GKE Version scanner
+
+GKE clusters running on older GKE versions can be exposed to security 
+vulnerabilities, or lack of support.  The GKE version scanner can ensure 
+your GKE clusters are running supported versions, and versions that are safe.
```

Comment:
More simply `... running safe and supported versions...`

Reply:
Done.